ShaderTool/Command/Material.cs

```diff
@@ -17,7 +17,7 @@ class MaterialData {
     class Material {
 
         public static string MaterialPath = Path.Combine(Program.ResourcesFolder, @"Materials.json");
-        public const uint MATERIAL_SIZE = 20; // Color is 4 * 4 bytes, diffuse texture ID is 4 bytes which equals 20 bytes
+        public const uint MATERIAL_SIZE = 24; // Color is 4 * 4 bytes, diffuse texture ID is 4 bytes which equals 20 bytes
```

Comment:
Why was this change made? Should update the comment as well

Reply:
This will be changed again with the new instance render system. Currently the animation index is also in the Fragment shader which is not needed. As described in the `UV offsetting per Fragment issue` in the Weekly log